micro-jackson-configuration/src/main/java/com/aol/micro/server/jackson/JacksonPlugin.java

```diff
@@ -6,26 +6,29 @@
 import com.fasterxml.jackson.databind.Module;
 import com.fasterxml.jackson.datatype.pcollections.PCollectionsModule;
 import cyclops.collections.immutable.PersistentSetX;
+import cyclops.collections.mutable.SetX;
+
+import java.util.Set;
 
 public class JacksonPlugin implements Plugin {
 
     @Override
-    public PersistentSetX<Class<?>> jaxRsResources() {
+    public Set<Class<?>> jaxRsResources() {
```

Comment:
We used persistent collection before, now it's mutable. Also do we still need import for PersistentSetX?

Reply:
Import no.  pCollections persistent collections also implement the j.u.Collection equivalent. In Cyclops X they don't (no impossible operations are allowed). Next Milestone will add j.u.Collection views and we can switch back.